### PR TITLE
ADAPT-4133: Trip Page a11y fixes

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- A couple of Trip Page a11y fixes (see things called out below). This should be the last of a11y fixes from the SODA team for T/S.

# Review By (Date)
- Dec 10 if possible, so we can cut a release that includes this work before holiday

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to https://deploy-preview-258--stanford-alumni.netlify.app/travel-study/trips/trip-preset-standard/
2. Scroll to Faculty Leader section below the video, check that the quote in the card is now a blockquote
Screen Shot 2021-12-08 at 1 57 33 PM

3. Scroll to the ankle, inspect the card headline, check that it's now a span instead of a heading. Check that it looks the same as before.
Screen Shot 2021-12-08 at 1 57 53 PM

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/components/cards/iconCard.js [hunk 1]
```diff
@@ -64,12 +63,7 @@ const IconCard = ({
           )}
           link={link}
         >
-          <Heading
-            level={headingLevel}
-            size="base"
-            weight="semibold"
-            className="su-mb-0"
-          >
+          <span className="su-block su-leading-display su-font-semibold">
```
Comment (yvonnetangsu) [inline src/components/cards/iconCard.js:66]: For Icon cards in the ankle, there is no need for the card headline to be a heading. I changed it to a `span`.

File: src/utilities/richTextRenderer.js [hunk 2]
```diff
@@ -126,6 +127,11 @@ const RichTextRenderer = ({ wysiwyg, isDark, className, linkColor }) => {
       [NODE_IMAGE]: (children, { src, alt }) => (
         <CardImage size="horizontal" filename={src} alt={alt} loading="lazy" />
       ),
+      [NODE_QUOTE]: (children) => (
+        <blockquote className="su-font-serif su-font-bold su-type-2 children:su-leading-cozy children:last:su-mb-06em">
+          {children}
+        </blockquote>
```
Comment (yvonnetangsu) [inline src/utilities/richTextRenderer.js:133]: Style blockquotes in Storyblok WYSIWYG - added that for horizontal basic card body field.

File: src/components/cards/iconCard.js [hunk 3]
```diff
@@ -2,15 +2,13 @@ import React from 'react';
...
 import { FlexBox } from '../layout/FlexBox';
-import { Heading } from '../simple/Heading';
 import SbLink from '../../utilities/sbLink';
 import HeroIcon from '../simple/heroIcon';
 import FaIcon from '../simple/faIcon';
 
 const IconCard = ({
   blok: {
     headline,
-    headingLevel = 3,
```
Comment (yvonnetangsu) [inline src/components/cards/iconCard.js:13]: I'll remove this schema in the Storyblok Icon Card component once this is merged and released to PROD

**Review by mattanglin — APPROVED** [review]
All lgtm! 👍🏼

File: src/utilities/richTextRenderer.js (hunk 2 above)
Comment (sherakama) [reply src/utilities/richTextRenderer.js:133]: Nice

**Review by sherakama — APPROVED** [review]

**Comment by yvonnetangsu** [comment]
Thank you both for the speedy review!

**Pull request merged**